developer_setup.md

```diff
@@ -26,6 +26,7 @@
   rpm -q --whatprovides npm || sudo dnf -y install npm   # For CentOS 7, Fedora 23 and older
   sudo dnf -y install openssl-devel                      # For rubygems
   sudo dnf -y install cmake                              # For rugged Gem
+  sudo dnf -y install openscap                           # For openscap Gem (the Gem installs without the lib but require fails)
```

Comment:
:+1: I would remove what's in the parenthesis as I think it is not interesting for someone setting up a development environment.